packages/schema-dot-org-json-ld-components/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "schema-dot-org-json-ld-components",
+  "version": "0.1.0",
+  "description": "Components to generate schema.org json-ld markup.",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --compilers js:babel-core/register"
```

Comment:
https://facebook.github.io/jest/docs/en/getting-started.html

The test files will need to match the default [testMatch](https://facebook.github.io/jest/docs/en/configuration.html#testmatch-array-string) pattern, or you can override that value if you don't want to rename the files. Other than that, the defaults should "just work", as it largely maps to the mocha API (`describe`, `it`, `beforeEach`, `afterEach`, etc).